lib/notification-issue.coffee

```diff
@@ -52,7 +54,10 @@ class NotificationIssue
       "#{repoUrl}/issues/new?title=#{@encodeURI(@getIssueTitle())}&body=#{@encodeURI(issueBody)}"
 
   getIssueTitle: ->
-    @notification.getMessage()
+    title = @notification.getMessage()
+    if title.length > TITLE_CHAR_LIMIT
+      title = title.substring(0, TITLE_CHAR_LIMIT - 3) + '...'
```

Comment:
Just a heads up, but you can use `\u2026` to make a 1 character length ellipsis since the `...` might look weird spacing wise in certain fonts.

http://www.fileformat.info/info/unicode/char/2026/index.htm